config/config.exs

```diff
@@ -50,8 +53,8 @@ config :fz_http,
   openid_connect_providers: "{}",
   saml_identity_providers: %{},
   saml_entity_id: "urn:firezone.dev:firezone-app",
-  saml_certfile_path: "apps/fz_http/priv/cert/saml_selfsigned.pem",
-  saml_keyfile_path: "apps/fz_http/priv/cert/saml_selfsigned_key.pem",
+  saml_certfile_path: Path.expand("../apps/fz_http/priv/cert/saml_selfsigned.pem", __DIR__),
```

Comment:
These are already overridden in `runtime.exs` but this will fix them for running any mix tasks within `apps/` in the `dev` or `test` environments.